### ISSUE TITLE
Action buttons are not announced in Explorer section title bar

### ISSUE THREAD

**Issue by stevencl** [Microsoft/vscode#3310]
Screen reader support turned on and using NVDA.
When focus is on the explorer section title bar (e.g., the title bar showing the name of the open folder) I do not hear the actions being announced as I tab through them. E.g., these actions aren't announced
![unannounced](https://cloud.githubusercontent.com/assets/1704059/13250636/12b65d4a-da23-11e5-9b82-e9435d230781.png)

Found when testing #3247

**Comment by bpasero** [comment]
Good catch, looks like the view-header has a role of button and thus cannot have buttons within. I changed the role to be "toolbar" now for the view header and it seems to make the buttons announce in NVDA and VoiceOver.
